README.md

```diff
@@ -39,12 +39,13 @@ $ scheduler --update-crontab
 ## Define your own job types
 ```php
 <?php
-Scheduler\job_type("pewpew", "cd :path && :task");
+Scheduler::push("pewpew", "cd :path && :task");
```

Comment:
I disagree the snake cased `job_type` but do agree the camel cased one `jobType`